jena-fuseki2/jena-fuseki-webapp/src/main/resources/org/apache/jena/fuseki/webapp/shiro.ini

```diff
@@ -19,6 +19,7 @@ admin=pw
 [urls]
 ## Control functions open to anyone
 /$/status  = anon
+/$/server  = anon
```

Comment:
Yes, because it affects non-localhost access so not directly JENA-2072 (it needs other URLs open) but noticed while working on 2072. "status" and "server" are synonyms (a polite way of says we ended up with both by accident) and go to the same servlet.